addons/mail/static/src/js/chat_manager.js

```diff
@@ -1120,6 +1126,21 @@ var ChatManager =  Class.extend(Mixins.EventDispatcherMixin, ServicesMixin, {
 
     get_channels_preview: function (channels) {
         var channels_preview = _.map(channels, function (channel) {
+            if (channel.channel_ids && _.contains(channel.channel_ids,"channel_inbox")) {
```

Comment:
@tivisse , To load inbox channel messages inside chat window and map inbox(mail_message) data with existing channel/chat template.